notebooks/examples/Identify Inspected Buildings.ipynb

```diff
@@ -0,0 +1,279 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Detect Inspected Buildings\n",
+    "\n",
+    "In damage assessment inspections, it's common that the structure needs to be fully inspected. One way to ensure that, is by having inspectors walk around the structure. In this example notebook, we'll how tracks can be used accomplish to identify structures that have been inspected. We'll use some sample data in Portland, ME to demonstrate this."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Import libraries and connect to the GIS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter password: ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "import arcgis\n",
+    "from datetime import datetime\n",
+    "\n",
+    "gis = arcgis.gis.GIS(\"https://dev0007109.esri.com/portal\", \"tracker_analysis\", verify_cert=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Fetch the building footprints\n",
+    "\n",
+    "The building footprints are stored in a hosted feature layer. Let's query them and create a spatially enabled dataframe. We are using UTM Zone 19 as the spatial reference in order to have more accurate results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sr ={'wkid': 32619, 'latestWkid': 32619}  # UTM Zone 19N\n",
+    "buildings_item = gis.content.get(\"c8aa37b7378641e79a0442b15efb359a\")\n",
+    "buildings_df = buildings_item.layers[0].query(as_df=True, out_sr=sr)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Create a map and plot the buildings"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<img src=></img>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "webmap = gis.map(\"150 Commercial Street, Portland, ME\", zoomlevel=18)\n",
+    "webmap.add_layer(buildings_df)\n",
+    "webmap"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Overlay some track data\n",
+    "Let's load some track data to visualize the problem."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<img src=></img>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "start_time = datetime(2020, 2, 1)\n",
```

Comment:
You could break out pendulum here if you wanted :)

`start_time = pendulum.datetime(2020, 2, 1, 0, 59, 59).format("YYYY-MM-DD HH:mm:ss", formatter='alternative')`